### PR TITLE
power BI learn section

### PR DESCRIPTION
**Pull request by isobelthompson**

### CODE REVIEW

File: src/data-visualisation/power-bi/learning/certifications/AI-exams/ai-index.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+collection_tag: ai-certs
+pagination:
+  data: collections.ai-certs
+  size: 50
+---
+There are many Microsoft Power Platforms and Azure certifications available. Please look at the pages available or the [attached spreadsheet][link] for more details.  
+  
+If you have any questions please get in touch with [Helen O'Donnell (Data Visualisation Manager)][email address].
+
+[link]: https://nhsbsauk.sharepoint.com/:x:/s/DataWarehouse/EccsayGlP-JDqSFfWK5BpXIBp3Ghr3wxMrjHUOtHmKQfCw?e=l1fvBH
```
Comment (WORTI3) [inline src/data-visualisation/power-bi/learning/certifications/AI-exams/ai-index.md:16]: Internal link only accessible by those within nhs bsa?

Comment (isobelthompson) [reply src/data-visualisation/power-bi/learning/certifications/AI-exams/ai-index.md:16]: Forgot about this one. Will have to discuss alternatives with HOD next week.

File: src/data-visualisation/power-bi/learning/certifications/AI-exams/ai-102.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+layout: page
+title: "AI-102"
+description: "Overview of the AI-102 exam"
+status: DRAFT
+tags: ai-certs
+order: 10
+---
+# Designing and Implementing a Microsoft Azure AI Solution Exam AI-102
```
Comment (WORTI3) [inline src/data-visualisation/power-bi/learning/certifications/AI-exams/ai-102.md:9]: Heading level looks wrong here should be `##`?

File: src/data-visualisation/power-bi/learning/certifications/AZ-exams/az-305.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+layout: page
+title: "AZ-305"
+description: "Overview of the AZ-305 exam"
+status: DRAFT
+tags: az-certs
+order: 30
+---
+# Designing Microsoft Azure Infrastructure Solutions Exam AZ-305
```
Comment (WORTI3) [inline src/data-visualisation/power-bi/learning/certifications/AZ-exams/az-305.md:9]: Heading level also looks wrong here

File: src/data-visualisation/power-bi/learning/certifications/AI-exams/ai-102.md (hunk 2 above)
Comment (isobelthompson) [reply src/data-visualisation/power-bi/learning/certifications/AI-exams/ai-102.md:9]: I set these all to # so they are larger, but can change to ## if necessary?

**Pull request merged**